server/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = false
```

Comment:
I would prefer a final new line. Otherwise, it can mess up output of `cat` and other CLI tools.